client/src/components/Progress.js

```diff
@@ -55,81 +57,105 @@ class Progress extends Component {
+                value={this.state.weight}
+                onChange={this.handleFieldChange}
+              />
+              <input
+                type="text"
+                name="hips"
+                placeholder="Hips"
+                value={this.state.hips}
+                onChange={this.handleFieldChange}
+              />
+              <input
+                type="text"
+                name="waist"
+                placeholder="Waist"
+                value={this.state.waist}
+                onChange={this.handleFieldChange}
+              />
+              <input
+                type="text"
+                name="r_arm"
+                placeholder="(R) Arm"
+                value={this.state.r_arm}
+                onChange={this.handleFieldChange}
+              />
+              <input
+                type="text"
+                name="l_arm"
+                placeholder="(L) Arm"
+                value={this.state.l_arm}
+                onChange={this.handleFieldChange}
+              />
+              <input
+                type="text"
+                name="r_leg"
+                placeholder="(R) Leg"
+                value={this.state.r_leg}
+                onChange={this.handleFieldChange}
+              />
+              <input
+                type="text"
+                name="l_leg"
+                placeholder="(L) Leg"
+                value={this.state.l_leg}
+                onChange={this.handleFieldChange}
+              />
+              <button className="Form__submit" type="submit">
+                Submit
+              </button>
+            </form>
+            <div className="progress-data">
+              <div className="current-weight" />
+              <div className="progress-records">
+                {this.props.progressRecords.map(record => {
+                  return (
+                    <div key={record._id} className="progress-record">
+                      <button
+                        onClick={() => this.props.deleteProgress(record._id)}
+                      >
+                        delete
+                      </button>
+                      <span>{`Weight: ${record.weight} lbs`}</span>
+                      <span>{`Hips: ${record.hips}in`}</span>
+                      <span>{`Waist: ${record.waist}in`}</span>
+                      <span>{`(R) Arm: ${record.r_arm}in`}</span>
+                      <span>{`(L) Arm: ${record.l_arm}in`}</span>
+                      <span>{`(R) Leg: ${record.r_leg}in`}</span>
+                      <span>{`(L) Leg: ${record.l_leg}in`}</span>
+                    </div>
+                  );
+                })}
+              </div>
+            </div>
+          </div>
+        </div>
       </div>
     );
   }
 }
 
+const mapStateToProps = state => {
+  return {
+    progressRecords: state.progress.progressRecords
+  };
+};
 
 Progress.propTypes = {
-    addProgress: PropTypes.func,
-}
-
-
+  addProgress: PropTypes.func,
```

Comment:
I'd like if you could talk about PropTypes tomorrow, so we can all start using it.